sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveMetadataCacheSuite.scala

```diff
@@ -126,4 +129,39 @@ class HiveMetadataCacheSuite extends QueryTest with SQLTestUtils with TestHiveSi
   for (pruningEnabled <- Seq(true, false)) {
     testCaching(pruningEnabled)
   }
+
+  test("cache TTL") {
+    val sparkConfWithTTl = new SparkConf().set(SQLConf.METADATA_CACHE_TTL.key, "1")
+    val newSession = SparkSession.builder.config(sparkConfWithTTl).getOrCreate().cloneSession()
+
+    withSparkSession(newSession) { implicit spark =>
+      withTable("test_ttl") {
+        withTempDir { dir =>
+          spark.sql(s"""
+            |create external table test_ttl (id long)
```

Comment:
nit: could you use uppercase for the SQL keywords , e.g., `CREATE EXTERNAL...`?

Reply:
Fixed in https://github.com/apache/spark/pull/28852/commits/7d94f943496fb463f18cd445ccbde3f5225c8e7b